multiapps-controller-process/src/main/java/org/cloudfoundry/multiapps/controller/process/steps/ServiceStep.java

```diff
@@ -61,6 +61,10 @@ private OperationExecutionState executeOperationAndHandleExceptions(ProcessConte
     protected abstract OperationExecutionState executeOperation(ProcessContext context, CloudControllerClient controllerClient,
                                                                 CloudServiceInstanceExtended service);
 
+    protected CloudOperationException createCloudOperationException(CloudOperationException e, String message, String... params) {
```

Comment:
This seems like an unnecessary method - it only calls one other method inside, with almost the same signature.